apisix/ssl.lua

```diff
@@ -55,23 +56,37 @@ function _M.server_name()
 end
 
 
-local _aes_128_cbc_with_iv = false
+local _aes_128_cbc_with_iv_tbl = false
 local function get_aes_128_cbc_with_iv()
-    if _aes_128_cbc_with_iv == false then
+    if _aes_128_cbc_with_iv_tbl == false then
+        _aes_128_cbc_with_iv_tbl = core.table.new(2, 0)
         local local_conf = core.config.local_conf()
-        local iv = core.table.try_read_attr(local_conf, "apisix", "ssl", "key_encrypt_salt")
-        if type(iv) =="string" and #iv == 16 then
-            _aes_128_cbc_with_iv = assert(aes:new(iv, nil, aes.cipher(128, "cbc"), {iv = iv}))
-        else
-            _aes_128_cbc_with_iv = nil
+        local ivs = core.table.try_read_attr(local_conf, "apisix", "ssl", "key_encrypt_salt")
+        local type_ivs = type(ivs)
+
+        if type_ivs == "table" then
+            for index, iv in ipairs(ivs) do
+                if type(iv) == "string" and #iv == 16 then
```

Comment:
Better to check this via schema in https://github.com/apache/apisix/blob/master/apisix/cli/schema.lua

Reply:
+1

Reply:
we need backward compatibility, how to support both table and string if use schema?

Reply:
Done